src/services/usersServer.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { User } = require('../models/user');
+
+function getAllUsers() {
+  return User.findAll({
+    order: ['created_at'],
+  });
+};
+
+function addUser(name) {
+  const newUser = User.create({ name });
+
+  return newUser;
+};
+
+function getUser(id) {
+  return User.findByPk(id);
+}
+
+function deleteUser(id) {
+  User.destroy({
+    where: {
+      id: id,
+    },
+  });
+
+  return true;
+}
+
+function updateUser(id, name) {
+  const newUser = User.update({
+    name,
+  }, {
+    where: { id }, returning: true,
+  });
+
+  return newUser;
+};
+
+function normalize({ id, name }) {
```

Comment:
looks redundant, if you need destruct your object you can write
`const {id, name} = user`

Reply:
done